index.html

```diff
@@ -0,0 +1,222 @@
+    <p>In contrast to Java's compile-time system of classes built by declarations, JavaScript supports a runtime system based on a small number of data types representing numeric, Boolean, and string values. JavaScript has a prototype-based object model instead of the more common class-based object model. The prototype-based model provides dynamic inheritance; that is, what is inherited can vary for individual objects. JavaScript also supports functions without any special declarative requirements. Functions can be properties of objects, executing as loosely typed methods.</p>
+    <p>JavaScript is a very free-form language compared to Java. You do not have to declare all variables, classes, and methods. You do not have to be concerned with whether methods are public, private, or protected, and you do not have to implement interfaces. Variables, parameters, and function return types are not explicitly typed.</p>
+</section>
+<section class="main-section" id="Hello_world">
+    <header>Hello world</header>
+    <p>To get started with writing JavaScript, open the Scratchpad and write your first "Hello world" JavaScript code:</p>
+    <code>function greetMe(yourName) { alert("Hello " + yourName); }
+    greetMe("World");</code>
+    <p>Select the code in the pad and hit Ctrl+R to watch it unfold in your browser!</p>
+</section>
+
+<section class="main-section" id="Variables">
+    <header>Variables</header>
+    <p>You use variables as symbolic names for values in your application. The names of variables, called identifiers, conform to certain rules.</p>
+    <p>A JavaScript identifier must start with a letter, underscore (_), or dollar sign ($); subsequent characters can also be digits (0-9). Because JavaScript is case sensitive, letters include the characters "A" through "Z" (uppercase) and the characters "a" through "z" (lowercase).</p>
+    <p>You can use ISO 8859-1 or Unicode letters such as å and ü in identifiers. You can also use the Unicode escape sequences as characters in identifiers. Some examples of legal names are Number_hits, temp99, and _name.</p>
+</section> 
+
+<section class="main-section" id="Declaring_variables">
+  <header>Declaring variables</header>
+    <p>You can declare a variable in three ways:</p>
+    <p>With the keyword var. For example,</p>
+    <code>var x = 42.</code>
+    <p>This syntax can be used to declare both local and global variables.</p>
+    <p>By simply assigning it a value. For example,</p>
+    <code>x = 42.</code>
+    <p>This always declares a global variable. It generates a strict JavaScript warning. You shouldn't use this variant.</p>
+    <p>With the keyword let. For example,</p>
+    <code>let y = 13.</code>
+    <p>This syntax can be used to declare a block scope local variable. See Variable scope below.</p>
+</section>
+
+<section class="main-section" id="Variable_scope">
+  <header>Variable scope</header>
```

Comment:
Replace your `header` element with heading element(h1, h2, h3, h4, h5, h6)

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear. eg why using the header as a substitute for your ```<hx>``` is not a good thing